rfcs/readme.md

```diff
@@ -0,0 +1,66 @@
+
+To create a new RFC you will:
+
+* Fork the OpenUI Design System repo - https://github.com/openui/design-system
+* Copy the `/rfcs/0000-template.md` file to `/rfcs/active` and rename it to `0000-proposed-component.md` where "proposed-component" is a descriptive title of your component.
+* Fill out the RFC template with your suggestions. Be sure to take care into filling out each portion of the template. **RFCs that do not conform to the template, demonstrate understanding of the imipact of the update, or are disingenous about the impact or alternatives tend to be poorly received**
+* Submit a pull request. Once it's a pull request, the RFC will receive feedback and discussion from the community. Don't worry about getting feedback, it serves to help make the process better. **Be prepared to update and revise your pull request based on community feedback.**
+* Build conensus around your RFC by engaging with the community and their feedback. RFCs with broad support are much more likely to be accepted than those without any comments.
+* RFCs which are candidates for inclusion will be tagged as such and enter a "final comment period" lasting no longer than a week.
+* Any RFC may be modified based on feedback from the community. Significant modifications may trigger a new "final comment period".
+* An RFC may be rejected by the community after public discussion has settled and the rationale for rejection has been provided in the comments on the pull request. Once this happens, the associated pull request will be closed by one of the moderators.
+* An RFC may be accepted at the close of the "final comment period" without any significant changes or challenges. At this point it will be merged in and become Active.
+
+## The RFC Lifecycle
+
+Once an RFC becomes merged and active, the authors may implement it and submit the feature as a pull request to the repo. Becoming 'active' is not a guarantee the method of implementation will be accepted, nor does it mean the feature will automatically be merged. It means the work can begin, but additional community feedback and revisions are likely.
+
+If during the process of implementing an RFC there are significant changes needed, the RFC can be re-opened in followup pull requests.
+
+Once the feature is code complete, documentation done, and testing provided, a final pull request will be created to officially merge the feature into the next release. Part of this pull request will also need to move the RFC from the Active directory (`/rfcs/active`) to the Accepted directory (`/rfcs/accepted`).
+
+## Implementing an RFC
+
+The author of the RFC does not have to be the person to implement it, however they should be ready to help once it is set to active if they would like to see it finally accepted. All community members are also welcome to help with the implementation of any active RFC.
```

Comment:
I'd prefer to use some stronger verbiage here, such as "does not have to be the person to implement it; however, we do recommend this approach as it helps speed up the RFC process."